var/spack/repos/builtin/packages/py-pbr/package.py

```diff
@@ -34,3 +34,4 @@ class PyPbr(PythonPackage):
     version('1.8.1', 'c8f9285e1a4ca6f9654c529b158baa3a')
 
     depends_on('py-setuptools', type='build')
+    depends_on('py-enum34', type='build')
```

Comment:
At least for python 2.7.13 and 3.3.6 ```enum``` is necessary. For 3.4.3 ```enum``` is no longer necessary. I don't know how to put this into the ```depends_on```-syntax, can you help me out?